view/frontend/web/template/payment/offsite-mobilebanking-form.html

```diff
@@ -0,0 +1,98 @@
+<div class="payment-method" data-bind="css: {'_active': (getCode() == isChecked())}, visible: isActive()">
+    <div class="payment-method-title field choice">
+        <input type="radio" name="payment[method]" class="radio" data-bind="attr: {
+               'id': getCode()
+               },
+               value: getCode(),
+                checked: isChecked,
+               click: selectPaymentMethod,
+               visible: isRadioButtonVisible(),
+               enable: isActive()" />
+        <label data-bind="attr: {'for': getCode()}" class="label">
+            <span data-bind="text: getTitle()"></span>
+        </label>
+        <div data-bind="visible: isSandboxOn()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <span data-bind="i18n: 'Test mode'"></span> (<a target="_blank"
+                        href="https://www.omise.co/what-are-test-keys-and-live-keys"><span
+                            data-bind="i18n: 'more info'"></span></a>)
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="payment-method-content">
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() -->
+        <!-- /ko -->
+        <!--/ko-->
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() -->
+            <!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="omise-banking-fee-warn"
+            data-bind="i18n: 'Your bank may charge a small fee for mobile banking payments'"></div>
+        <form class="form" data-bind="attr: {
+              id: getCode() + 'Form',
+              }
+              ">
+
+            <fieldset data-bind="attr: {class: 'fieldset payment items ' + getCode(), id: 'payment_form_' + getCode()}">
+                <ul class="form-list">
+                    <!-- SCB -->
+                    <li class="items" data-bind="visible: isAllowCurrency(['thb'])">
```

Comment:
can you confirm if `isAllowCurrency` will hide these payment methods for Singapore merchants? Just making sure since mobile banking methods are only offered locally

Reply:
i provided examples image on above 👆 is come with example checkout currency

Reply:
ahh yea I see it now, all good!